pkg/model/pipeline.go

```diff
@@ -237,8 +235,16 @@ func LoadPipelineDefintionFromOneLiner(oneLine []string) (Definition, error) {
 
 		conf.Masking = append(conf.Masking, masking)
 	}
-	if conf.Seed == 0 {
+	SetSeed(&conf, conf.Seed)
+	return conf, nil
+}
+
+// setSeed compute seed from current nano second if seedValue is 0
+func SetSeed(conf *Definition, seedValue int64) {
```

Comment:
This could be a func attached to the `model.Definition` object.

```suggestion
func (conf *Definition) SetSeed(seedValue int64) {
```